README.md

```
@@ -8,21 +8,98 @@
+## 🚀 Live Demo
+
+Check out the live application here: [skynet1.netlify.app](https://skynet1.netlify.app)
+
+---
+
+## 🛠️ Local Setup Instructions
+
+Follow these steps to run the project locally:
+
+1. **Clone the repository**:
+   ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)